packages/botonic-plugin-nlu/src/utils.js

```diff
@@ -0,0 +1,121 @@
+import fs from 'fs'
```

Comment:
els fitxers de utils poden acabar sent un calaix de sastre.

sense coneixer massa el codi una opció seria fer un fitxer per la persistencia(load/save) i un per coses realment util/generiques q s'usin desde varius llocs (eg. clone, replaceAll)

Reply:
i'll clean this by creating a utils.js (with the generics methods you suggest) and then another one for persistance (maybe fileUtils.js could be a good name?).

Reply:
fileUtils is a classic 👍

Reply:
done!